### PR TITLE
XZ documentation setup

### PR DESCRIPTION
**Pull request by xiyaozhuang**
This pull request adds the necessary documentation for setting up the dependencies for this repository.
- Synthea International
- Large Language Models
- Entity extraction
- CorrectMatch

### CODE REVIEW

File: docs/setup/index.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    Required to clone this repository as well as [Synthea](https://github.com/synthetichealth/synthea) and [Synthea International](https://github.com/synthetichealth/synthea-international).
+
+    [:material-arrow-right: Git](https://git-scm.com/)
+
+-   :simple-python: __Python__
+
+    ---
+
+    This project is built using Python 3.11.7 release.
```
Comment (danjscho) [inline docs/setup/index.md:19]: Do we need to give more the 3.11?

File: docs/setup/llm.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Large Language Models
+
+!!! warning
+
+    Requires [Ollama](https://github.com/ollama/ollama) to run. This particular setup was tested with [Ollama version 0.1.27](https://ollama.com/download) on an M1 MBP.
+
+[Ollama](https://github.com/ollama/ollama) is used for the unstructured generative component of Privacy Fingerprint. It provides a simple interface to download quantised models and run inference locally.
+
+## Start Ollama
```
Comment (danjscho) [inline docs/setup/llm.md:9]: Do we bluntly say "Install ollama" anywhere?

**Review by danjscho — APPROVED** [review]
Couple of minor questions which might need some minor additions if you want, but looks good.

Also, double checked, the new version of flake8 in the `pre-commit-config` is fine for 3.11.

File: docs/setup/llm.md (hunk 2 above)
Comment (xiyaozhuang) [reply docs/setup/llm.md:9]: there's a big warning at the top that says it requires Ollama to run - i think this is sufficient

File: docs/setup/index.md (hunk 1 above)
Comment (xiyaozhuang) [reply docs/setup/index.md:19]: probably not - will change to 3.11

**Pull request merged**